source/fuzz/fact_manager.cpp

```diff
@@ -588,6 +588,22 @@ void FactManager::DataSynonymAndIdEquationFacts::AddEquationFactRecursive(
   // Now try to work out corollaries implied by the new equation and existing
   // facts.
   switch (opcode) {
+    case SpvOpConvertFToS:
+    case SpvOpConvertFToU: {
```

Comment:
I guess there should be an analogous pair of cases for when the opcode is OpCovert[S|U]ToF?

Reply:
Indeed, fixed in e4b2f8c.